pkg/config/v1alpha3/types.go

```diff
@@ -148,6 +148,7 @@ type SimpleConfig struct {
 	Image           string                  `mapstructure:"image" yaml:"image,omitempty" json:"image,omitempty"`
 	Network         string                  `mapstructure:"network" yaml:"network,omitempty" json:"network,omitempty"`
 	Subnet          string                  `mapstructure:"subnet" yaml:"subnet,omitempty" json:"subnet,omitempty"`
+	HostPidMode     bool                    `mapstructure:"host-pid-mode" yaml:"host-pid-mode,omitempty" json:"host-pid-mode,omitempty"`
```

Comment:
I feel like this option should be nested unter `simpleConfig.Options.Runtime.HostPidMode`, what do you think?

Reply:
Thank you for your comment :bow:
I agree with you! I will fix it!

Reply:
I have fixed!!
https://github.com/rancher/k3d/pull/929/commits/586c71dbbb944da6375e20ed01646533a2a8aab7